javafx-archetype-simple/src/main/resources/archetype-resources/pom.xml

```diff
@@ -6,15 +6,22 @@
     <version>${version}</version>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+#if ("8" == ${javafx-version})
+            <maven.compiler.source>8</maven.compiler.source>
+            <maven.compiler.target>8</maven.compiler.target>
+#else
+            <maven.compiler.source>11</maven.compiler.source>
+            <maven.compiler.target>11</maven.compiler.target>
+#end
     </properties>
     <dependencies>
+#if ("8" != ${javafx-version})
```

Comment:
I'd consider using profiles here. We should encourage users to migrate to 11...

As I [commented](https://github.com/openjfx/javafx-maven-plugin/pull/44#issuecomment-538893933) in the plugin PR, imo, adding a profile in case you want to run with 11+ is better than needing to modify the whole pom because your project only runs on 8 to 10.

Reply:
See https://github.com/openjfx/javafx-maven-archetypes/pull/8 for this.